src/prashant/Facebook.java

```diff
@@ -59,5 +59,6 @@ public static void main(String[] args) throws InterruptedException {
 			System.out.println("You are still on same page");
 		}		
 		driver.navigate().refresh();
```

Comment:
Only One suggestion, after refreshing if we are not checking anything then what is the advantage of refreshing page at the end of the script.